tests/lib/core-config.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+#shellcheck source=tests/lib/systems.sh
+. "$TESTSLIB"/systems.sh
+
+clean_snapd_lib() {
+    rm -rf /var/lib/snapd/assertions/*
+    rm -rf /var/lib/snapd/device
+    rm -rf /var/lib/snapd/state.json
+}
+
+prepare_model(){
+    mv /var/lib/snapd/seed/assertions/model model.bak
+}
+
+restore_model(){
+    cp model.bak /var/lib/snapd/seed/assertions/model
```

Comment:
Why `cp` and not move? If this runs without a prior call to `prepare_model` then the stale data will be used.

Reply:
done, thanks for the review